test/NuGet.Core.FuncTests/Dotnet.Integration.Test/DotnetRestoreTests.cs

```diff
@@ -892,6 +892,56 @@ public async Task DotnetRestore_MultiTargettingWithAliases_Succeeds()
             }
         }
 
+        [PlatformFact(Platform.Windows)]
```

Comment:
Is there a reason this is windows only?

Reply:
All of the tests in here are windows only.

Reply:
umm, no they're not?

https://github.com/NuGet/NuGet.Client/blob/0b2568615d61e9f9393d597cb806f02af1b40a78/test/NuGet.Core.FuncTests/Dotnet.Integration.Test/DotnetRestoreTests.cs#L86

![image](https://user-images.githubusercontent.com/5030577/89696530-25112f80-d8cd-11ea-9da1-7e650b1a1fc1.png)

Reply:
I stand corrected. 
11/14 are windows only. 

I can use a regular fact, that's not a big deal.